backend/api/communication/src/main/java/co/airy/core/api/communication/dto/CountAction.java

```diff
@@ -1,6 +1,31 @@
 package co.airy.core.api.communication.dto;
 
-public enum CountAction {
-    INCREMENT,
-    RESET
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@NoArgsConstructor
+public class CountAction implements Serializable {
+    public enum ActionType {
+        INCREMENT,
+        RESET
+    }
+
+    private long readDate;
+    private ActionType actionType;
+
+    CountAction(ActionType actionType, long readDate) {
```

Comment:
`@AllArgsConstructor` does not give us this? 🤔

Reply:
yeah good point! I'll update